### PR TITLE
Use []byte instead of MsgID for identifier in qbft

### PR DESCRIPTION
**Pull request by nkryuchkov**

### CODE REVIEW

File: automation/qbft/scenarios/sync_failover.go [hunk 1]
```diff
@@ -146,7 +146,8 @@ loop:
 	}
 	r.logger.Info("node #4 synced", zap.Int64("highest decided", int64(nextSeq)-1))
 
-	decides, err := ctx.Stores[3].GetDecided(message.ToMessageID(msgs[1].Message.Identifier), 0, nextSeq)
+	messageID := message.ToMessageID(msgs[1].Message.Identifier)
```
Comment (nivBlox) [inline automation/qbft/scenarios/sync_failover.go:149]: @nkryuchkov 
we need to change the `msgs[1].Message` to use spec message struct

File: exporter/api/query_handlers.go [hunk 2]
```diff
@@ -39,7 +40,7 @@ func HandleDecidedQuery(logger *zap.Logger, qbftStorage qbftstorage.QBFTStore, n
 	identifier := spectypes.NewMsgID(pkRaw, message.RoleTypeFromString(string(nm.Msg.Filter.Role)))
```
Comment (nivBlox) [inline exporter/api/query_handlers.go:40]: lets rename identifier to msgID

**Review by nivBlox — CHANGES_REQUESTED** [review]

File: automation/qbft/scenarios/sync_failover.go (hunk 1 above)
Comment (nkryuchkov) [reply automation/qbft/scenarios/sync_failover.go:149]: @nivBlox Do you mean to change `*specqbft.SignedMessage` to `*specqbft.Message`?

Comment (nivBlox) [reply automation/qbft/scenarios/sync_failover.go:149]: @nkryuchkov yes

File: exporter/api/query_handlers.go (hunk 2 above)
Comment (nkryuchkov) [reply exporter/api/query_handlers.go]: done

**Pull request merged**